zebra-consensus/src/transaction.rs

```diff
@@ -298,6 +302,29 @@ where
         )
     }
 
+    /// Verifies if a V4 `transaction` is supported by `network_upgrade`.
+    fn verify_v4_transaction_network_upgrade(
+        transaction: &Transaction,
+        network_upgrade: NetworkUpgrade,
+    ) -> Result<(), TransactionError> {
+        match network_upgrade {
+            // Supports V4 transactions
```

Comment:
Just finishing off #2480
```suggestion
            // Supports V4 transactions
            //
            // Consensus rules:
            // > [Sapling to Canopy inclusive, pre-NU5] The transaction version number MUST be 4, ...
            // >
            // > [NU5 onward] The transaction version number MUST be 4 or 5.
            //
            // https://zips.z.cash/protocol/protocol.pdf#txnconsensus
```